src/Utils/Utils.jl

```diff
@@ -508,34 +508,12 @@ function matrix_conversion(Δ, options; A_passed=nothing)
     if create_new_system
         # set new matrix
         if hasmethod(inputset, Tuple{typeof(Δ.s)})
```

Comment:
I know this was not part of this PR, but do we maybe have a better way to check whether a system has inputs? Maybe we can create an issue.

Reply:
One "general" way is to check if `inputdim(Δ) > 0`. I've opened https://github.com/JuliaReach/Reachability.jl/issues/513 for this.